admin-manual/maintenance/cli-import-export.rst

```diff
@@ -1910,7 +1919,7 @@ The following example import command will:
 
 .. code-block:: bash
 
-   php symfony csv:physicalobject-import --update --skip-unmatched --multi-match="all" --partial-match --skip-rows=10 --error-log="/usr/share/nginx/atom/errors.log" --index /path/to/my/storage.csv
+   php symfony csv:physicalobject-import --update --skip-unmatched --multi match="all" --partial-match --skip-rows=10 --error-log="/usr/share/nginx/atom/errors.log" --index /path/to/my/storage.csv
```

Comment:
hmm, was this an intentional change? the dash between multi and match has been removed...

Reply:
No, ended up getting the chop in the process of "undo-ing" code-block spits.. Fixed it.